docs/pages/_app.tsx

```diff
@@ -30,9 +32,11 @@ class MyApp extends App<{ theme: ThemeType }> {
     const { Component, pageProps, theme } = this.props;
 
     return (
-      <PageWithContexts initialTheme={theme} pageContext={this.pageContext}>
-        <Component pageContext={this.pageContext} {...pageProps} />
-      </PageWithContexts>
+      <ReduxProvider store={store}>
```

Comment:
Please wrap only the particular example in redux provider. So redux will be included in bundle only on the forms page